index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;
+    0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;
+    1,700;1,800&display=swap" rel="stylesheet">
+    <title>Nyalik Portfolio</title>
+</head>
+<body>
+    <nav>
+        <ul class="toolbar-container">
+            <li class="logo">Nyalik Marv</li>
```

Comment:
- [ ] As required for the project, kindly ensure that the logo is a link (i.e. it uses a `<a>` tag). This is for three reasons: `convention`, `semantics`, and user `experience`(UX). You can [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.